config/sageone.yml

```diff
@@ -1,17 +1,22 @@
 sageone:
   client_id: 'YOUR_CLIENT_ID'
   client_secret: 'YOUR_CLIENT_SECRET'
-  signing_secret: 'YOUR_SIGNING_SECRET'
   apim_subscription_key: 'YOUR_APIM_SUBSCRIPTION_KEY'
-  callback_url: 'YOUR_CALLBACK_URL'
-  auth_endpoint: 'https://www.sageone.com/oauth2/auth'
+  callback_url: 'http://localhost:3000/auth/callback'
```

Comment:
Better use a value that makes it clear a change is required

Reply:
There is no change required. `http://localhost:3000/auth/callback` is the configured callback URL of this application.